### PR TITLE
Safely ignore ResizeObserver error

### PR DESCRIPTION
**Pull request by terryyylim**
This PR attempts to fix the following issue.

image

Related issues:
- https://github.com/atlassian/react-beautiful-dnd/issues/1548
- https://github.com/atlassian/react-beautiful-dnd/issues/2223

### CODE REVIEW

**Comment by krithika369** [comment]
Thanks, @terryyylim. The fix LGTM. Before merging, let's test it out on the internal repo.

File: package.json [hunk 1]
```diff
@@ -53,7 +53,8 @@
     "build:flow": "echo \"// @flow\n\nexport * from '../src';\" > dist/react-beautiful-dnd.cjs.js.flow",
     "storybook": "start-storybook -p 9002",
     "build-storybook": "build-storybook -c .storybook -o site",
-    "prepublishOnly": "yarn build"
+    "prepublishOnly": "yarn build",
+    "prepare": "yarn build:clean && yarn build:dist && yarn build:flow"
```
Comment (krithika369) [inline package.json:57]: So, this `prepare` script is automatically run on `yarn install` to create the `dist` folder?

Comment (terryyylim) [reply package.json:57]: Yes that is correct!

Comment (terryyylim) [reply package.json:57]: Dropping a link here for reference: https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts

**Review by krithika369 — APPROVED** [review]

**Pull request merged**